srcpkgs/vit/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 task python3-pytz python3-urwid python3-tzlocal python3-tasklib"
+checkdepends="python3-devel task python3-tzlocal python3-urwid python3-tasklib"
+short_desc="Visual Interactive Taskwarrior full-screen terminal interface"
+maintainer="Cullen Ross <cullenrss@gmail.com>"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+distfiles="${PYPI_SITE}/v/vit/vit-${version}.tar.gz"
+checksum=fd34f0b827953dfdecdc39f8416d41c50c24576c33a512a047a71c1263eb3e0f
+
+pre_check() {
+	export TERM=linux
+	export TERMINFO=/usr/share/terminfo
+}
```

Comment:
```suggestion
do_check() {
	TERM=linux python3 -m pytest
}
```

Reply:
Thanks! I wasn't sure what the default do_check() function was, so this change makes waaaay more sense.